css-three-column-layout/styles.css

```diff
@@ -0,0 +1,74 @@
+.container{
+  max-width: 1000px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.row{
+  display: flex;
+}
+
+.blue-background{
+  background-color: #1896d9;
+}
+
+.column-one-third{
+  flex-basis: calc(100%/3);
+}
+
+.column-two-thirds{
+  flex-basis: calc(100%/3);
+}
+
+.column-third{
+  flex-basis: calc(100%/3);
+}
+
+.justify-center{
+  padding: 5px;
+  justify-content: center;
+}
+
+.justify-center h3, p{
+  padding: 5px;
+  justify-content: center;
+}
```

Comment:
You should check if the `justify-content` declaration here is necessary. If the property does nothing, it should be removed.